cylc/flow/task_remote_mgr.py

```diff
@@ -230,6 +230,9 @@ def remote_tidy(self):
             if install_target == get_localhost_install_target():
                 continue
             platform = get_random_platform_for_install_target(install_target)
+            if platform is None:
+                LOG.debug(f"Remote tidy not required for {install_target}.")
+                continue
```

Comment:
Not sure we want to silently fail here?

This could just mean that none of the platforms or hosts are accessible.